react/common/utils/date.ts

```diff
@@ -1,9 +1,14 @@
-
 export function getApiDateFormat(date: Date) {
-
-    const day = ("0" + (date.getDate())).slice(-2);
-    const month_number = ("0" + (date.getMonth() + 1)).slice(-2)
+    const day = ('0' + date.getDate()).slice(-2);
+    const month_number = ('0' + (date.getMonth() + 1)).slice(-2);
```

Comment:
Not a big deal but we should ensure every variable is in `camelCase` in the Javascript codebase, could you rename this one `monthNumber`?

Reply:
Did it too 👍